README.md

```diff
@@ -1,7 +1,7 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_center_spider_DOM/)
+    - [DEMO LINK](https://SerhiiSemennikov.github.io/js_center_spider_DOM/)
```

Comment:
Add demo link to the PR description 
<img width="973" alt="image" src="https://github.com/user-attachments/assets/0cc31aa9-6032-4dbf-9539-ea337c972840">